### PR TITLE
Change default font size in web apps forms to 16px

### PR DESCRIPTION
**Pull request by AddisonDunn**
## Product Description

This PR changes the base font size in web apps FORMS from 12px to 16px. Affects form labels and input font size.

In response to [this ticket](https://dimagi.atlassian.net/browse/USH-4054?atlOrigin=eyJpIjoiMDk3MWZhODdlMzIzNGQ4YTgyMmQ1NTQyOGZhYTY4ZDciLCJwIjoiaiJ9).

## Technical Summary

Client wants to decrease information density.

### Safety story

- Test locally and on staging
- Can only affect CSS/font size

### Rollback instructions

- [X] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [X] Risk label is set correctly
- [X] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

**Review by MartinRiese — APPROVED** [review]

**Comment by nospame** [comment]
Know we're probably a while from a Bootstrap 5 migration for web apps, but does the B5 style sheet (I think [_variables.scss](https://github.com/dimagi/commcare-hq/blob/ad/change-web-app-font-size/corehq/apps/hqwebapp/static/hqwebapp/scss/commcarehq/_variables.scss)) need to be updated as well?

File: corehq/apps/hqwebapp/tests/data/bootstrap5_diffs/stylesheets/imports/includes_variables._variables.style.diff.txt [hunk 1]
```diff
@@ -16,7 +16,7 @@
 +  xxl: 1160px
 +);
  
--@font-size-base:  12px;
+-@font-size-base:  14px;
```
Comment (orangejenny) [inline corehq/apps/hqwebapp/tests/data/bootstrap5_diffs/stylesheets/imports/includes_variables._variables.style.diff.txt:19]: I think this change should also happen in B5. As-is, the font will shrink back down when we migrate.

File: corehq/apps/hqwebapp/static/hqwebapp/less/_hq/includes/variables.less [hunk 2]
```diff
@@ -3,7 +3,7 @@
 // Nunito Sans is used on dimagi.com and embedded in hqwebapp/base.html
 @font-family-sans-serif: "Nunito Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
 
-@font-size-base:  12px;
+@font-size-base:  14px;
```
[the remark this answers is not in the document]
Comment (biyeun) [reply corehq/apps/hqwebapp/static/hqwebapp/less/_hq/includes/variables.less:6]: yes this will affect all of HQ

**Comment by biyeun** [comment]
i think our font size is too small tbh, that seems fine

**Comment by biyeun** [comment]
I would check its effect on reports, forms, app builder, and vellum

**Review by nospame — APPROVED** [review]
This change looks good to me. Can you add a screenshot(s) here showing the new appearance?

**Pull request merged**